### PR TITLE
Set warning messages separately for category,language and Skill name

### LINKED ISSUE #3067: Display proper message when skill details are not provided
**Actual Behaviour**

Currently the alert message which displays when no skill_name is given to the skill is `Please select a group, language and a skill`.

**Expected Behaviour**

It should just prompt the user to give name to the skill.

**Screenshots of the issue**

immm

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Dilshaad21**
Fixes #3067 

Changes: 
Instead of hardcoding the warning message when no `category`,`language` or `skill name` is provided changes are done such that it displays warning messages accordingly.
 
Demo Link: https://pr-3068-fossasia-susi-web-chat.surge.sh

Screenshots of the change:
 
vvdv

### CODE REVIEW

File: src/components/cms/SkillCreator/SkillWizard.js [hunk 1]
```diff
@@ -717,9 +717,25 @@ class SkillWizard extends Component {
...
     }
-    if (category === null || language === '' || name === '') {
+    if (name === '') {
       this.props.actions.openSnackBar({
-        snackBarMessage: 'Please select a group, language and a skill',
+        snackBarMessage: 'Please give a skill name',
+        snackBarPosition: { vertical: 'top', horizontal: 'right' },
+        variant: 'warning',
+      });
+      return 0;
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillWizard.js]: It involves a lot of redundant code, instead write a function that returns the missing field.

**Comment by Dilshaad21** [comment]
@akshatnitd done the changes.

File: src/components/cms/SkillCreator/SkillWizard.js [hunk 2]
```diff
@@ -679,6 +679,19 @@ class SkillWizard extends Component {
     });
   };
 
+  giveWarningMessage = (language, category, name) => {
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillWizard.js]: Rename to `getSnackbarMessage`

File: src/components/cms/SkillCreator/SkillWizard.js [hunk 3]
```diff
@@ -679,6 +679,19 @@ class SkillWizard extends Component {
...
+    let message = '';
+    if (language === '') {
+      message += 'language ';
+    }
+    if (category === null) {
+      message += 'category ';
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillWizard.js]: Use an array for storing the fields which is empty.

File: src/components/cms/SkillCreator/SkillWizard.js [hunk 4]
```diff
@@ -679,6 +679,19 @@ class SkillWizard extends Component {
...
+    }
+    if (name === '') {
+      message += 'name ';
+    }
+    return message;
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillWizard.js]: map through the array and insert `,` in between. 
And then return the entire message including `Please enter ...`.

Eg message - `Please do not leave the category and skill fields empty`.

**Pull request merged**